src/main/msp/msp.c

```diff
@@ -1909,6 +1907,12 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, currentPidProfile->thrustLinearization);
 #else
         sbufWriteU8(dst, 0);
+#endif
+        // Added 20201_04_25
+#if defined(USE_INTERPOLATED_SP)
+    sbufWriteU8(dst, currentPidProfile->ff_jitter_factor);
+#else
+    sbufWriteU8(dst, 0);
```

Comment:
Comment not needed. There's already a comment further up for stuff added in 1.44.
Needs more spaces 🙂